contents/docs/libraries/go/index.mdx

```diff
@@ -77,108 +77,47 @@ client.Enqueue(posthog.Capture{
+    Event:      "event_name",
+    Properties: map[string]interface{}{
+        "$set": map[string]interface{}{
+            "name": "Max Hedgehog",
+        },
+        "$set_once": map[string]interface{}{
+            "initial_url": "/blog",
+        },
+    },
 })
 ```
 
-**Usage**
-
-`$set_once` works just like `$set`, except that it will **only set the property if the user doesn't already have that property set**.
-
-### Identify
-
-> We highly recommend reading our section on [Identifying users](/docs/integrate/identifying-users) to better understand how to correctly use this method.
-
-Identify lets you add metadata to your users so you can easily identify who they are in PostHog, as well as do things
-like segment users by these properties.
-
-An identify call requires:
-
--   `distinct id` which uniquely identifies your user
--   `properties` with a dictionary of any key:value pairs you'd like to add
-
-For example:
-
-```go
-client.Enqueue(posthog.Identify{
-  DistinctId: "user:123",
-  Properties: posthog.NewProperties().
-    Set("email", "john@doe.com").
-    Set("proUser", false),
-})
-```
-
-The most obvious place to make this call is whenever a user signs up, or when they update their information.
+For more details on the difference between `$set` and `$set_once`, see our [user properties docs](/docs/data/user-properties#what-is-the-difference-between-set-and-set_once).
 
 ### Alias
 
-To connect whatever a user does before they sign up or login with what they do after, you need to make an alias call. This will allow you to answer questions like "Which marketing channels lead to users churning after a month?" or "What do users do on our website before signing up?"
-
-In a purely back-end implementation, this means whenever an anonymous user does something, you'll want to send a session ID with the capture call. Then, when that users signs up, you want to do an alias call with the session ID and the newly created user ID.
-
-The same concept applies for when a user logs in.
-
-If you're using PostHog in the front-end and back-end, doing the identify call in the frontend will be enough.
+> We strongly recommend reading our docs on [alias](/docs/integrate/identifying-users#alias-assigning-multiple-distinct-ids-to-the-same-user) to better understand how to correctly use this method.
 
-An alias call requires
-
--   `DistinctId` – the user id
--   `Alias` – the anonymous session distinct ID
-
-For example:
+Sometimes, you may want to assign multiple distinct IDs to a single user. This is helpful in scenarios where your primary distinct ID may be inaccessible. For example, if a distinct ID which is typically used on the frontend is not available in certain parts of your backend code. In this case, you can use `alias` to assign another distinct ID to the same user:
 
 ```go
 client.Enqueue(posthog.Alias{
-  DistinctId: "user:123",
-  Alias: "session:12345",
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
```

Comment:
lots of folks got it wrong, when it was like this in the past (used an already identified id as alias_id) - that's why I changed it to be `session:12345` and `user:123` to make it clear that the session id needs to be non-identified user. Suggestion: don't write any code or anything about this into specific library docs but link to the alias section instead.

Reply:
I like this suggestion. I'll implement it